### PR TITLE
Update the script so we always build the image before composing when …

### PR DESCRIPTION
**Pull request by PeterJausovec**
…Release compose file is used.

### CODE REVIEW

File: generators/app/templates/_dockerTaskGeneric.sh [hunk 1]
```diff
@@ -38,8 +38,13 @@ compose () {
   else
     echo "Running compose file $composeFileName"
     docker-compose -f $composeFileName kill
-    docker-compose -f $composeFileName up -d
     
+    if [[ $ENVIRONMENT = "release" ]]; then
```
Comment (NCarlsonMSFT) [inline generators/app/templates/_dockerTaskGeneric.sh]: Why do we only always build for release?

Comment (NCarlsonMSFT) [inline generators/app/templates/_dockerTaskGeneric.sh]: If you don't change the behavior, please do change the usage

Comment (PeterJausovec) [inline generators/app/templates/_dockerTaskGeneric.sh]: The idea is that in debug we use volume sharing to share the source code inside of the container - we don't have to rebuild the image each time as the changes are automatically made inside of the container. For release, we'd want to re-build the image as we don't want to use the cached instance of it.

File: generators/app/templates/_dockerTaskGeneric.sh [hunk 2]
```diff
@@ -64,7 +69,7 @@ showUsage () {
     echo ""
     echo "Commands:"
     echo "    build: Builds a Docker image ('$imageName')."
-    echo "    compose: Runs docker-compose."
+    echo "    compose: Builds the images and runs docker-compose. Images are re-built when using release environment while debug environment uses a cached version of the image."
```
Comment (NCarlsonMSFT) [inline generators/app/templates/_dockerTaskGeneric.sh]: This run-on needs some punctuation. Minimum add a comma after release environment.

**Pull request merged**